src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -72,6 +74,19 @@ object ECDSASignature {
     else s
   }
 
+  def sign(message: Array[Byte], keyPair: AsymmetricCipherKeyPair): ECDSASignature = {
```

Comment:
Probably, though it's kinda strange this wasn't there from the start. This PR already does several things beyond its planned scope so I'd rather not add even more to it. 

There's actually a couple of things to be refactored about the `ECDSASignature`, IMO. I will make task of that add a note about the test. Pinging @LukasGasior1